source/install-atlas-cli.txt

```diff
@@ -210,77 +210,126 @@ Follow These Steps
+                        {+mdbVersion+}.list`` for your version of
+                        Ubuntu. Replace ``{+mdbVersion+}`` with your
+                        edition of MongoDB.
+
+                        .. tabs::
+
+                           .. tab:: Ubuntu 22.04 (Jammy)
+                              :tabid: comm-jammy
+
+                              .. code-block:: sh
+
+                                 echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu jammy/mongodb-org/{+mdbVersion+} multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-{+mdbVersion+}.list
+
+                           .. tab:: Ubuntu 20.04 (Focal)
+                              :tabid: comm-focal
+
+                              .. code-block:: sh
+
+                                 echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu focal/mongodb-org/{+mdbVersion+} multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-{+mdbVersion+}.list
+
+                           .. tab:: Ubuntu 18.04 (Bionic)
+                              :tabid: comm-bionic
 
-                        .. code-block:: sh
+                              .. code-block:: sh
 
-                           echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu focal/mongodb-org/5.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-5.0.list
+                                 echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu bionic/mongodb-org/{+mdbVersion+} multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-{+mdbVersion+}.list
 
-                     .. tab:: Ubuntu 18.04 (Bionic)
-                        :tabid: comm-bionic
+                     .. tab:: Debian
+                        :tabid: debian
 
-                        .. code-block:: sh
+                        Create the list file 
+                        ``/etc/apt/sources.list.d/mongodb-org-
+                        {+mdbVersion+}.list`` for your version of
+                        Debian. Replace ``{+mdbVersion+}`` with your
+                        edition of MongoDB.
 
-                           echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu bionic/mongodb-org/5.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-5.0.list
+                        .. tabs::
 
-                     .. tab:: Debian 10 (Buster)
-                        :tabid: comm-deb-10
+                           .. tab:: Debian 10 (Buster)
+                              :tabid: comm-deb-10
 
-                        .. code-block:: sh
+                              .. code-block:: sh
 
-                           echo "deb http://repo.mongodb.org/apt/debian buster/mongodb-org/5.0 main" | sudo tee /etc/apt/sources.list.d/mongodb-org-5.0.list
+                                 echo "deb http://repo.mongodb.org/apt/debian buster/mongodb-org/{+mdbVersion+} main" | sudo tee /etc/apt/sources.list.d/mongodb-org-{+mdbVersion+}.list
```

Comment:
**Question**: Same space issue as above, with `debian` and `buster`, though this looks intentional now that I see it on one line. Still have to ask because that seems unusual for a path.

Reply:
Both should be correct based on existing docs